programs/benchmark/Benchmark.cpp

```diff
@@ -601,6 +601,9 @@ int mainEntryClickHouseBenchmark(int argc, char ** argv)
     {
         using boost::program_options::value;
 
+        const char * env_user = getenv("CLICKHOUSE_USER");
```

Comment:
clang-tidy is saying that `getenv` implementation permitted to rewrite shared buffer (although it does not happen on Linux).

Reply:
is there a better way of doing this? I was following the [code from `clickhouse-client` binary](https://github.com/ClickHouse/ClickHouse/blob/master/programs/client/Client.cpp#L179).

Reply:
After subsequent `getenv` call, the result of previous call is invalidated.
Solution: copy it to std::string.

Reply:
Thanks for suggestions